engine/dlib/src/dlib/zip.h

```diff
@@ -84,6 +84,11 @@ namespace dmZip
      *
      */
     Result GetEntryData(HZip zip, void* buffer, uint32_t buffer_size);
+
+    /*# gets partial data for the currently open entry
+     *
+     */
+    Result GetEntryDataOffset(HZip zip, uint32_t offset, uint32_t size, void* buffer, uint32_t* nread);
```

Comment:
This will be used to stream sounds form a .zip file, while minimizing the RAM overhead